charts/ontology/README.md

```diff
@@ -0,0 +1,79 @@
+DUOS Ontology
+========
+A Helm chart for DUOS Ontology, the DUOS Algorithmic Matching System
+
+Current chart version is `0.1.0`
+
+
+
+
+
+## Chart Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| global.applicationVersion | string | Is set by Helmfile on deploy | Ontology global version |
+| appVersion | string | Is set by Helmfile on deploy | Ontology image version/tag. Required unless using `image`. |
+| devDeploy | bool | `false` | Enable to deploy to dev locally with Skaffold |
+| elasticSearch.server1 | string | `nil` | first elastic search host |
+| elasticSearch.server2 | string | `nil` | second elastic search host |
+| elasticSearch.server3 | string | `nil` | third elastic search host |
+| environment | string | `nil` | Environment, [dev, staging, prod] |
+| gcsAccount.path | string | `nil` | Vault path to GCS base64 service account json |
```

Comment:
There are often good reasons to break this rule (have done it many times myself and lots of our charts have nested values), but just as an FYI, flat values are preferred to nested ones in official Helm best practices: https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values